src/auth/jwt.strategy.ts

```diff
@@ -15,4 +15,8 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
       secretOrKey: configService.get('JWT_SECRET'),
     });
   }
+  async validate(payload: { email: string; role: string }) {
+    return { email: payload.email, role: payload.role };
+  }
 }
+export default JwtStrategy;
```

Comment:
Not used

Reply:
I can’t find something related when I try to delete it returns errors